contrib/syntax.yml

```diff
@@ -228,4 +228,11 @@ rust:
     open: '/*\n'
     close: ' */\n\n'
     prefix: ' * '
-
+# Conf files i.e. apache config, splunk.conf files
+conf:
+  ext:  ['.conf']
+  after: ['^#!']
```

Comment:
the shebang is probably not relevant for the configs. Probably just make this an empty array (`[]`)

Reply:
Good point